src/RTCPeerConnection.ts

```diff
@@ -157,15 +166,23 @@ export default class RTCPeerConnection extends defineCustomEventTarget(...PEER_C
             desc = null;
         }
 
+        this.pendingLocalDescription = new RTCSessionDescription(desc);
```

Comment:
LEt's store this in some `_pendingLocalDescription` and add a getter, so it's not settable.